docs/reference/nuspec.md

```diff
@@ -139,7 +139,35 @@ license-expression =  1*1(simple-expression / compound-expression / UNLICENSED)
 ```
 
 #### iconUrl
+
+> [!Important]
+> licenseUrl is being deprecated. Use license instead.
+
 A URL for a 64x64 image with transparency background to use as the icon for the package in UI display. Be sure this element contains the *direct image URL* and not the URL of a web page containing the image. For example, to use an image from GitHub, use the raw file URL like <em>https://github.com/\<username\>/\<repository\>/raw/\<branch\>/\<logo.png\></em>. 
+   
+#### icon
+Path to an image file within the package, often shown in UIs like nuget.org as the package icon. Image file size is limited to 1 MB. Supported file formats include JPEG and PNG. We recommend an image resoulution of 64x64.
```

Comment:
A markdown linter would tell you to add an empty line between the heading the first line in the paragraph.